### PR TITLE
dev-util/packer: version bump to 1.5.6

### PR DESCRIPTION
**Pull request by mrgz**
Package-Manager: Portage-2.3.103, Repoman-2.3.23
Signed-off-by: Marcos Rodrigues Gonzalez <mrgonzalez1275@gmail.com>

### CODE REVIEW

File: dev-util/packer/packer-1.5.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit go-module
+
+DESCRIPTION="A tool to create identical machine images for multiple platforms"
+HOMEPAGE="https://www.packer.io"
```
Comment (juippis) [inline dev-util/packer/packer-1.5.6.ebuild:9]: Please also add the Github page here as a 2ndary homepage.

File: dev-util/packer/packer-1.5.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (juippis) [inline dev-util/packer/packer-1.5.6.ebuild:1]: You should update this to 1999-2020. You can commit with a tool called `repoman` that does this automatically for you.

File: dev-util/packer/packer-1.5.6.ebuild (hunk 1 above)
Comment (mrgz) [reply dev-util/packer/packer-1.5.6.ebuild:9]: Do you have some examples?

**Comment by mrgz** [comment]
I'll do it all over

**Pull request closed without merging**